metaflow/plugins/env_escape/stub.py

```diff
@@ -381,7 +394,10 @@ def create_class(
             name = name[7:]
             method_type = CLASS_METHOD
         if name in overriden_methods:
-            if method_type == NORMAL_METHOD:
+            if name == "__init__":
+                class_dict["__init_overriden__"] = overriden_methods["__init__"]
```

Comment:
if we're doing this, why not go all the way and allow overriding all the dunder methods?

Reply:
I think we may actually already be doing it. The only reason init was special is because it doesn't go through call/callattr. I think the other dunders should go with NORMAL_METHOD below. One thing though is that we may want to restrict it to some whitelisted ones but I feel that this is not a huge problem since this is a very internal thing.